app/src/main/java/org/simple/clinic/main/TheActivity.kt

```diff
@@ -271,7 +271,7 @@ class TheActivity : AppCompatActivity(), TheActivityUi {
     screenRouter.registerKeyChanger(FullScreenKeyChanger(
         activity = this,
         screenLayoutContainerRes = android.R.id.content,
-        screenBackgroundRes = R.color.window_background,
```

Comment:
Do we need to change this since we are going to be removing this class once we finish migrating to the new navigation framework?

Reply:
This was just so I can remove all the older colors. It's not actually required but paves way for me to add the `resolveColor` method that can fetch color for a given theme attribute.